### PR TITLE
2.6.x blog post

### PR DESCRIPTION
**Pull request by IAlibay**
Mostly just copying across from the release notes I wrote earlier.

### CODE REVIEW

**Comment by IAlibay** [comment]
pinging @fiona-naughton if you wanted to take a stab at cleaning this up.

File: _posts/2023-08-16-release-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: Releases 2.6.0 of MDAnalysis
+---
+
+We are happy to release the version 2.6.0 of MDAnalysis!
```
Comment (RMeli) [inline _posts/2023-08-16-release-2.6.0.md:6]:
```suggestion
We are happy to release version 2.6.0 of MDAnalysis!
```
?

File: _posts/2023-08-16-release-2.6.0.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+made with NumPy 1.25+ which offers backwards runtime compatibility
+with NEP29 supported version of NumPy.
+
+Supported Python versions: **3.9, 3.10, 3.11**
+
+Supported Operating Systems:
+  - **Linux** (on [x86-64][], [aarch64][])
+    - *Note: aarch64 can only be installed via pip for now*
+  - **Windows** (on [x86-64][])
+  - **MacOS** (on [x86-64][] and [Apple M1][])
```
Comment (RMeli) [inline _posts/2023-08-16-release-2.6.0.md:26]:
```suggestion
  - **MacOS** (on [x86-64][] and [ARM64][])
```
?

File: _posts/2023-08-16-release-2.6.0.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+To update with `conda` from the [conda-forge channel][] run
+
+```bash
+conda update -c conda-forge mdanalysis
+```
+
+To update from [PyPi][] with `pip` run
+
+```bash
+pip install --upgrade MDAnalysis
```
Comment (RMeli) [inline _posts/2023-08-16-release-2.6.0.md:40]:
```suggestion
python -m pip install --upgrade MDAnalysis
```

File: _posts/2023-08-16-release-2.6.0.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+
+#### Deprecations:
+ - The misspelt `Boltzman_constant` entry in `MDAnalysis.units` is now
+   deprecated in favour the correctly spelt `Boltzmann_constant`. (PRs #4230 and #4214)
+ - `MDAnalysis.analysis.hole2` is now deprecated in favour of a new
+   [HOLE2 MDAKit](https://www.mdanalysis.org/hole2-mdakit/). (PR #4200)
+
+## Author statistics
+
+This release was the work of 7 contributors, **3** of which are **new contributors.**
```
Comment (RMeli) [inline _posts/2023-08-16-release-2.6.0.md:98]:
```suggestion
This release was the work of 7 contributors, **3** of which are **new contributors**.
```

File: _posts/2023-08-16-release-2.6.0.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Release 2.6.1 of MDAnalysis
+---
+
+We are happy to release version 2.6.1 of MDAnalysis!
```
Comment (orbeckst) [inline _posts/2023-08-16-release-2.6.0.md:6]:
```suggestion
We are happy to release version 2.6.1 of MDAnalysis!
(This blog post describes 2.6.1 and the initial release in the 2.6 series, 2.6.0.)
```
Just to make clear that all the changes below really belong to 2.6.0 without going into details (which I don't know) about why we needed 2.6.1 right away... I assume it's numpy and conda-forge...

File: _posts/2023-08-16-release-2.6.0.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title: Release 2.6.1 of MDAnalysis
```
Comment (orbeckst) [inline _posts/2023-08-16-release-2.6.0.md:3]:
```suggestion
title: Release 2.6.0 and 2.6.1 of MDAnalysis
```

**Comment by IAlibay** [comment]
Thanks for the review @orbeckst !

File: _posts/2023-08-30-release-2.6.0.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+
+[x86-64]: https://en.wikipedia.org/wiki/X86-64
+[aarch64]: https://en.wikipedia.org/wiki/AArch64
+[ppc64le]: https://en.wikipedia.org/wiki/Ppc64
+[ARM64]: https://en.wikipedia.org/wiki/Apple_M1
+[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
+[conda-forge channel]: https://anaconda.org/conda-forge/mdanalysis
+[PyPi]: https://pypi.org/project/MDAnalysis/
+[NumFOCUS]: https://www.numfocus.org
+[CHANGELOG]: https://github.com/MDAnalysis/mdanalysis/blob/release-2.6.0/package/CHANGELOG
```
Comment (fiona-naughton) [inline _posts/2023-08-30-release-2.6.0.md:121]:
```suggestion
[CHANGELOG]: https://github.com/MDAnalysis/mdanalysis/blob/release-2.6.1/package/CHANGELOG
```

File: _posts/2023-08-30-release-2.6.0.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+## Acknowledgements
+
+MDAnalysis thanks [NumFOCUS][] for its continued support as our fiscal sponsor and the [Chan Zuckerberg Initiative][] for supporting MDAnalysis under an EOSS4 award.
+
+— The MDAnalysis Team
+
+
+[x86-64]: https://en.wikipedia.org/wiki/X86-64
+[aarch64]: https://en.wikipedia.org/wiki/AArch64
+[ppc64le]: https://en.wikipedia.org/wiki/Ppc64
```
Comment (fiona-naughton) [inline _posts/2023-08-30-release-2.6.0.md:115]: Apologies for my ignorance here, but we have a link here for ppc64 but it's not used in the post itself (though is mentioned in previous release posts) - is this intentional? (Not a huge thing and wouldn't let this stop releasing the blog post, but just wanted to check)

Comment (IAlibay) [reply _posts/2023-08-30-release-2.6.0.md:115]: Yeah we should just remove the link - so we used to occasionally test on ppc64le, and could claim that it was supported, but we fully lost access to Travis CI lately and can't test it. We do release conda-forge wheels for it still, we just don't really know if it works.

Comment (fiona-naughton) [reply _posts/2023-08-30-release-2.6.0.md]: ok - thanks! I'll remove it just to keep things cleaner

**Review by fiona-naughton — APPROVED** [review]
Sorry for taking a while to look through - added some very minor updates, but should be ready to go now :smile_cat:

File: _posts/2023-09-02-release-2.6.1.md [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+
+For a full list of changes, bugfixes and deprecations see the [CHANGELOG][].
+
+
+#### Fixes:
+ - Fix to element guessing code to more accurately interpret atom names split by
+   numbers (i.e. N0A is now recognised as N rather than NA) [Issue #4167, PR #4168]
+ - The -ffast-math compiler flag is no longer used by default at build time,
+   avoiding inconsistent (although still scientifically correct) results seen
+   in Intel MacOS systems when calling `AtomGroup.center_of_charge(..., unwrap=True).
```
Comment (orbeckst) [inline _posts/2023-09-02-release-2.6.1.md:55]:
```suggestion
   in Intel MacOS systems when calling `AtomGroup.center_of_charge(..., unwrap=True)`.
```
adding close backtic

**Review by orbeckst — COMMENTED** [review]
minor fix, otherwise great, thanks @fiona-naughton & everyone else ... found after merge, so will fix on live

**Pull request merged**